src/templates/partials/header.njk

```diff
@@ -1,10 +1,16 @@
 <header>
-    <h1>Site Header</h1>
-    <nav>
-        <ul>
-            <li><a href="/">Home</a></li>
-            <li><a href="/about">About</a></li>
-            <li><a href="/contact">Contact</a></li>
-        </ul>
-    </nav>
+  <a href="{{ baseUrl }}/index.html">
+    <img src="{{ baseUrl }}/src/images/static/binabox-logo.webp" alt="Binabox Logo" />
+  </a>
+  <nav>
+    <ul>
```

Comment:
I suggest use cycle here for links. And you need to add condition. 
If page url = href, then add modifier class to a link.

Reply:
Find example in presentation or watch recordings of our lecturers.
All links available here https://github.com/bro-academy/HTML-course-4/blob/main/lesson-8-bundlers.md